src/config/index.ts

```diff
@@ -120,4 +121,8 @@ export const APP_CONFIG: AppConfig = {
     component: Analytics(),
     eventCb,
   },
+
+  // Provide friendly names and optional icons for addresses that should be rendered
```

Comment:
nit, remove this comment (redundant with the `types.ts` def), and want to keep all the docs there